server/model/unlawfullyAtLargeForm.ts

```diff
@@ -60,9 +60,30 @@ export default class UnlawfullyAtLargeForm extends AdjustmentsForm<UnlawfullyAtL
       errors.push(toDateError)
     }
 
+    const fromDate = new Date(
```

Comment:
Do you need to have this on the frontend? The validation already happens on the backend. I think the approach with validation has been to do as much as possible on the API. The frontend only validates that the data is the correct type in order to pass to the backend, i.e. a valid date or valid integer

The javascript date stuff isn't great! If this needs to stay can you write some tests for it?

Reply:
The wider MOJ approach for forms is to validate input fields on the UI if possible (e.g. where no further api calls are necessary), this idea is that you dont make unnecessary calls to API's.  Therefore have added this here as well. Good call regards to tests. Will add some